wrapper/minecraft/src/main/kotlin/org/quiltmc/qkl/wrapper/minecraft/serialization/internal/SerializerCodec.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qkl.wrapper.minecraft.serialization.internal
+
+import com.mojang.datafixers.util.Pair
+import com.mojang.serialization.Codec
+import com.mojang.serialization.DataResult
+import com.mojang.serialization.DynamicOps
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.modules.SerializersModule
+import org.quiltmc.qkl.wrapper.minecraft.serialization.internal.decoder.DynamicDecoder
+import org.quiltmc.qkl.wrapper.minecraft.serialization.internal.encoder.DynamicEncoder
+import org.quiltmc.qkl.wrapper.minecraft.serialization.options.CodecOptions
+
+internal class SerializerCodec<A>(
```

Comment:
This class should have toString implemented for debugging purposes, as is it just returns the jvm representation

![image](https://user-images.githubusercontent.com/52360088/196085367-401503eb-842a-4185-a25f-ba42916037da.png)